include/pnet_api.h

```diff
@@ -1213,10 +1209,11 @@ typedef struct pnet_ip_cfg
  */
 typedef struct pnet_if_cfg
 {
-   pnet_netif_t main_port; /**< Main (DAP) network interface. */
+   pnet_netif_name_t main_netif; /**< Main (DAP) network interface. */
```

Comment:
```suggestion
   const char * network_interface_name; /**< Main (DAP) network interface. */
```
(Assuming that pnet_cfg_t is no longer stored in pnet_t.)

Reply:
will use "netif_name"